src/fidesops/service/connectors/manual_connector.py

```diff
@@ -0,0 +1,62 @@
+from typing import Any, Dict, List, Optional
+
+from fidesops.common_exceptions import PrivacyRequestPaused
+from fidesops.graph.traversal import TraversalNode
+from fidesops.models.policy import ActionType, Policy
+from fidesops.models.privacy_request import PrivacyRequest
+from fidesops.service.connectors.base_connector import DB_CONNECTOR_TYPE, BaseConnector
+from fidesops.util.collection_util import Row
+
+
+class ManualConnector(BaseConnector[None]):
+    def query_config(self, node: TraversalNode) -> None:
+        """No query_config for the Manual Connector"""
+        return None
+
+    def create_client(self) -> DB_CONNECTOR_TYPE:
+        """Not needed because this connector involves a human performing some lookup step"""
+        return None
+
+    def close(self) -> None:
+        """No session to close for the Manual Connector"""
+        return None
+
+    def test_connection(self) -> None:
+        """No automated test_connection available for the Manual Connector"""
+        return None
+
+    def retrieve_data(
+        self,
+        node: TraversalNode,
+        policy: Policy,
+        privacy_request: PrivacyRequest,
+        input_data: Dict[str, List[Any]],
+    ) -> Optional[List[Row]]:
+        """
+        Returns manually added data for the given collection if it exists, otherwise pauses the Privacy Request.
+        """
+        cached_results: Dict[
+            Optional[str], Optional[List[Row]]
+        ] = privacy_request.get_manual_input(node.address)
+
+        if cached_results:
+            privacy_request.cache_paused_step_and_collection()  # Caches paused location as None
+            return list(cached_results.values())[0]
```

Comment:
Well the current design only allows one node to be paused at any given time now that we're running the nodes sequentially.  

Separately, I think I still want the [0] because I'm trying to get the corresponding value out of the list of dictionary values, I want to remove the outer wrapper:

```python
cached_results
{'EN_MANUAL_INPUT__test_postgres_access_request_task_586cddf4-2694-4437-8192-fdadbf87430f__manual_example:storage_unit': [{'box_id': 5, 'email': 'customer-1@example.com'}]}

cached_results.values()
dict_values([[{'box_id': 5, 'email': 'customer-1@example.com'}]])

list(cached_results.values())[0]
[{'box_id': 5, 'email': 'customer-1@example.com'}]
```